interleave.el

```diff
@@ -2,7 +2,8 @@
 
 ;; Author: Sebastian Christ <rudolfo.christ@gmail.com>
 ;; URL: https://github.com/rudolfochrist/interleave
-;; Version: 0.2.0
```

Comment:
The pull request is comparing against a slightly older version. So I am missing some `git` step. Let me know if this is obvious to you. What do I need to do once you have merged the pull request and both of our "master"s are in sync? Do I need to hard reset my fork to your master (How?)?